modules/cli/src/main/scala/coursier/cli/options/ResolutionOptions.scala

```diff
@@ -9,58 +9,85 @@ import coursier.parse.{DependencyParser, ModuleParser, ReconciliationParser, Rul
 
 // format: off
 final case class ResolutionOptions(
-
+  @Group(OptionGroup.resolution)
+  @Hidden
   @Help("Keep optional dependencies (Maven)")
     keepOptional: Boolean = false,
 
+  @Group(OptionGroup.resolution)
+  @Hidden
   @Help("Maximum number of resolution iterations (specify a negative value for unlimited, default: 100)")
   @Short("N")
     maxIterations: Int = ResolutionProcess.defaultMaxIterations,
 
+  @Group(OptionGroup.resolution)
   @Help("Force module version")
   @Value("organization:name:forcedVersion")
   @Short("V")
     forceVersion: List[String] = Nil,
 
+  @Group(OptionGroup.resolution)
+  @Hidden
   @Help("Set property in POM files, if it's not already set")
   @Value("name=value")
     pomProperty: List[String] = Nil,
 
+  @Group(OptionGroup.resolution)
+  @Hidden
   @Help("Force property in POM files")
   @Value("name=value")
     forcePomProperty: List[String] = Nil,
 
+  @Group(OptionGroup.resolution)
+  @Hidden
   @Help("Enable profile")
   @Value("profile")
     profile: List[String] = Nil,
 
+  @Group(OptionGroup.resolution)
   @Help("Default scala version")
   @Short("e")
   @Short("scala")
     scalaVersion: Option[String] = None,
 
+  @Group(OptionGroup.resolution)
```

Comment:
I would hide this one. That's a pretty arcane option.

Reply:
Done